doc/source/guide/export.rst

```diff
@@ -293,6 +293,67 @@ If you would like to output HTML be returned directly via :code:`save_as_html`,
 
 The HTML export functionality is the basis for exporting to interactive data apps, described next.
 
+Exporting to DataPane
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`DataPane <https://datapane.com/>`_ is a Python library used for creating interactive data science reports. To integrate Lux with a DataPane report, you can wrap the HTML widget output with a `dp.HTML <https://docs.datapane.com/reports/blocks/text-code-and-html#html>`_ block as shown below:
+
+.. code-block:: python
+
+  import pandas as pd
+  import datapane as dp
+  import lux
+  df = pd.read_csv("https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/hpi.csv")
+  html_content = df.save_as_html(output=True)
+  dp.Report("## Analysis of Happy Planet Index Dataset\n Check out these cool visualizations!",
+    dp.HTML(html_content)
+  ).save(path='report.html', open=True)
+
+You should find that a webpage :code:`report.html` that pops up automatically.
+
+
+.. image:: https://github.com/lux-org/lux-resources/blob/master/doc_img/export-datapane.png?raw=true
+  :width: 700
+  :align: center
+
+Exporting to Panel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Panel <https://panel.holoviz.org/>`_ is a powerful Python library for interactive data exploration and data apps. To integrate Lux with Panel, you can wrap the HTML widget output with a `panel.pane.HTML <https://panel.holoviz.org/reference/panes/HTML.html>`_ pane as shown below:
+
+.. code-block:: python
+
+  import html
+
+  import pandas as pd
+  import panel as pn
+  import lux
+
+  pn.extension(sizing_mode="stretch_width")
+
+  df = pd.read_csv("https://raw.githubusercontent.com/lux-org/lux-datasets/master/data/hpi.csv")
+
+  def to_lux_panel(df, style="width:100%;height:100%", frameborder="0"):
+      html_content = df.save_as_html(output=True)
+      html_content = html.escape(html_content)
+      iframe_report = f"""<iframe srcdoc="{html_content}" style={style} frameborder="{frameborder}"
+  "allowfullscreen></iframe>"""
+
+      return pn.pane.HTML(iframe_report, height=425)
+
+  # Can display in notebook
+  lux_panel = to_lux_panel(df)
+
+  # Can display as app with panel serve
+  pn.template.FastListTemplate(site="💡 Lux and Panel", title="Analysis of Happy Planet Index Dataset", main=[lux_panel]).servable();
+
+After running :code:`panel serve app.ipynb` or :code:`panel serve app.py` in the command line, you should find the Panel app hosted in your browser (e.g., :code:`http://localhost:5006/app`):
+
+
+.. image:: https://user-images.githubusercontent.com/42288570/122705585-56c98580-d256-11eb-9bd6-a259cc2af5d6.png
```

Comment:
This image should be added to `lux-resources` and the url updated.

Reply:
I've just added the image to lux-resources. Please replace line 353 with 
```
.. image:: https://github.com/lux-org/lux-resources/blob/master/doc_img/export-panel.png?raw=true
```

Reply:
Done @micahtyong